website/src/pages/Projects/styles.css

```diff
@@ -1,3 +1,10 @@
 div#projects-page img {
     max-height: 80vh;
 }
+div#projects-page .container {
+    display: grid;
+    gap: 8rem;
+}
+div#projects-page figure > img.logo {
+    max-width: 200px;
+}
```

Comment:
I'm a bit annoying with this but please be careful to always leave a newline at the end of your files. Every time you make a pull request please double-check the files view to see the github diff before asking a review. This view will show you this small red warning if you have files without an extra newline at the end. We can talk about it if you don't know why this is even a thing :wink:

Reply:
It should be ok now.